guides/source/testing.md

```diff
@@ -507,7 +516,17 @@ You should test for things such as:
 * was the correct object stored in the response template?
 * was the appropriate message displayed to the user in the view?
 
-Now that we have used Rails scaffold generator for our `Article` resource, it has already created the controller code and tests. You can take look at the file `articles_controller_test.rb` in the `test/controllers` directory.
+Now that we have used Rails scaffold generator for our `Article` resource,
+it has already created the controller code and tests. You can take
+look at the file `articles_controller_test.rb` in the `test/controllers` directory.
+
+The following command will generate a controller test case with a filled up
+test for each of the seven default actions.
+
+```bash
+$ rails g test_unit:scaffold article
```

Comment:
Could you add the "bin/" prefix here and use "generate" instead of "g" to be consistent please ?